packages/react-scripts/scripts/init.js

```diff
@@ -265,9 +280,10 @@ module.exports = function(
     return;
   }
 
-  if (tryGitInit(appPath)) {
+  // Create git commit if git repo was initialized
+  if (initializedGit && tryGitCommit(appPath)) {
```

Comment:
I'd probably break out `tryGitCommit` into another line and call it `didCommit` or something... but I think it's OK like this too.